<launch>
	<arg name="veh"/>
    <arg name="map_name" default="tiles_lab" />
	<arg name="config" default="baseline"/>
	<arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman"/>
	<arg name="local" default="false" doc="true to launch locally on laptop. false to launch of vehicle"/>
	<arg name="pkg_name" default="fleet_planning" doc="name of the package"/>
	<arg name="node_name" default="graph_search_server_node" doc="name of the node"/>
	<arg name="map_dir" default="$(find fleet_planning)/src/maps/"/>
    <param name="map_name" value="$(arg map_name)" />
    <param name="veh" value="$(arg veh)"/>
	<param name="map_dir" value="$(arg map_dir)"/>

	<group ns="$(arg veh)">
	    <!-- Local -->
	    <node if="$(arg local)" pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen" clear_params="true" required="true">
	    </node>

	    <!-- Remote -->
		<include unless="$(arg local)" file="$(find duckietown)/machines"/>
	    <node unless="$(arg local)" machine="$(arg veh)" pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen" clear_params="true" required="true">
	    </node>
	</group>
</launch>
